crates/core/src/client/mod.rs

```diff
@@ -631,6 +631,23 @@ impl KakarotClient for KakarotClientImpl<JsonRpcClient<HttpTransport>> {
         Ok(Address::from(evm_address_sliced))
     }
 
+    /// Get the nonce for a given ethereum address
+    /// Convert Ethereum address to Starknet address
+    /// Return the nonce, by calling `get_nonce` on the account contract
```

Comment:
Update comments to reflect new refactor and lgtm

Reply:
Ah, nice catch!

Done [here](https://github.com/kkrt-labs/kakarot-rpc/pull/181/commits/a370e83eb4344dbd77b2b2b1591ea8599945302e).